addons/data_recycle/static/src/views/data_recycle_list_view.js

```diff
@@ -3,21 +3,15 @@
 import { DataCleaningCommonListController } from "@data_recycle/views/data_cleaning_common_list";
 import { registry } from '@web/core/registry';
 import { listView } from '@web/views/list/list_view';
-import { session } from "@web/session";
 
 export class DataRecycleListController extends DataCleaningCommonListController {
     /**
      * Validate all the records selected
      */
     async onValidateClick() {
-        let record_ids;
-        if (this.isDomainSelected) {
-            const domain = this.props.domain;
-            record_ids = await this._domainToResIds(domain, session.active_ids_limit);
-        } else {
-            record_ids = await this.getSelectedResIds();
-        }
-
+        // _domainToResIds is legacy ? It should crash.
+        // getSelectedResIds check isDomainSelected
+        const record_ids = await this.getSelectedResIds();
```

Comment:
This is yannick's code so I am not sure, I would guess `isDomainSelected` never exists.
Your diff looks ok

Reply:
I think it's legacy code. The model didn't work exactly the same way I think.